tests/validation/tests/v3_api/common.py

```diff
@@ -1,3 +1,4 @@
+from ..common import *  # NOQA
```

Comment:
Will all files referencing the deleting methods in here work? because we added this line now? Or should this line be made available in the other files too?